components/ButtonBar/src/MDCButtonBarButton.h

```diff
@@ -0,0 +1,26 @@
+//
+//  MDCButtonBarButton.h
+//  MaterialComponents
+//
+//  Created by Ian Gordon on 1/17/18.
+//
+
+#import "MaterialButtons.h"
+
+@interface MDCButtonBarButton : MDCFlatButton
+
+// Content padding for the button.
```

Comment:
I believe this should be /** */ comment style.

Reply:
Done.